components/mintmaker/production/base/manager_patches.yaml

```diff
@@ -15,3 +15,6 @@ spec:
           requests:
             cpu: 100m
             memory: 256Mi
+        env:
+        - name: RENOVATE_IMAGE
+          value: "quay.io/konflux-ci/mintmaker-renovate-image:067c096e9e0e7de05041eb0fe27f329d8c1c993cb8c73e00637981d299408b2d"
```

Comment:
The current image being used in production is a newer one (`40af0570a79be63de3487670e1d6172ba00b7dcc`) released yesterday. To improve traceability and ease of reference, I recommend using the git commit hash tag instead of the SHA digest hash. The git commit hash tag directly indicates which commit the image corresponds to, making it more intuitive to track changes and debug issues.

Reply:
I agree